src/foam/u2/view/ReadOnlyEnumView.js

```diff
@@ -9,39 +9,69 @@ foam.CLASS({
   name: 'ReadOnlyEnumView',
   extends: 'foam.u2.View',
 
+  requires: ['foam.u2.tag.CircleIndicator'],
+
+  imports: ['theme'],
+
   css: `
     ^{
       border-radius: 11.2px;
+      border: 1px solid;
       font-family: /*%FONT1%*/ Roboto, 'Helvetica Neue', Helvetica, Arial, sans-serif;
       font-size: 10px;
       font-stretch: normal;
       font-style: normal;
       font-weight: 500;
       letter-spacing: normal;
-      line-height: 24px;
+      line-height: 2.1em;
       min-width: 60px;
       padding: 0 12px;
       text-align: center;
       width: -webkit-max-content;
       width: -moz-max-content;
+      display: inline-flex;
+      justify-content: space-around;
+      align-items: center;
+    }
+    ^icon{
+      margin-right: 4px;
     }
   `,
 
   documentation: 'Creates badges with rounded/squared sides based on display context',
 
+  properties: [
+    {
+      class: 'Boolean',
+      name: 'showGlyph'
+    }
+  ],
+
   methods: [
-    function initE(data) {
+    function initE() {
+      var data = this.data;
       this.SUPER();
+      var icon;
```

Comment:
Does icon need to be declared here? The variable is only used within the .callIf

Reply:
Not really, will move it to the call statement